sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryPipelineRequest.java

```diff
@@ -52,16 +53,16 @@ public String getInstrumentationKey() {
     HttpRequest createHttpRequest() {
         HttpRequest request = new HttpRequest(HttpMethod.POST, url);
         request.setBody(Flux.fromIterable(telemetry));
-        request.setHeader("Content-Length", Integer.toString(contentLength));
+        request.setHeader(HttpHeaderName.fromString("Content-Length"), Integer.toString(contentLength));
```

Comment:
let's make constants for these `HttpHeaderName` values (I _think_ that's the reason for prefering this over passing in String)